cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/TransactionHttpHandler.java

```diff
@@ -175,4 +208,78 @@ public void resetTxManagerState(HttpRequest request, HttpResponder responder) {
     txClient.resetState();
     responder.sendStatus(HttpResponseStatus.OK);
   }
+
+  @Path("/transactions/pruneinfo")
+  @POST
```

Comment:
If it is a GET call, then the region name would have to be specified in the path of the request. So wondering if those characters might conflict with the HTTP request path/syntax.